apps/client/src/components/compounds/transactionHistory/TransactionHistory.tsx

```diff
@@ -0,0 +1,75 @@
+import { Card, Text } from 'components';
+import styles from './transactionHistory.module.scss'
+import { useState } from 'react';
+
+const table = [ 
+    {
+        id: 1,
+        receiver: 'Bolade Adegbete',
+        type: 'Lagos Corp Group',
+        date: '30 May 2023',
+        amount: '5000'
+    },
+    {
+        id: 2,
+        receiver: 'Bolade Adegbete',
+        type: 'Lagos Corp Group',
+        date: '30 May 2023',
+        amount: '5000'
+    },
+    {
+        id: 3,
+        receiver: 'Bolade Adegbete',
+        type: 'Lagos Corp Group',
+        date: '30 May 2023',
+        amount: '5000'
+    },
+    
+]
+
+const TransactionHistory = () => {
+    const [selected, setSelected] = useState<number | null>(null);
+
+    const handleClick = (index: number) => {
+        setSelected(index);
+      };
+    
+
+  return (
+    <div  className={styles.container}>
+      <div className={styles.containerHeader}>
+      <Text content={'TRANSACTION HISTORY'} size={'Small'} />
+      <div className={styles.select}>
+      <Text content={'Recently'} size={'Small'} className={` ${selected === 0 ? 'selected' : ''} ${styles.button}`} onClick={() => handleClick(0)}/>
+      <Text content={'Oldest'} size={'Small'}  className={`button ${selected === 2 ? 'selected' : ''} ${styles.button}`} onClick={() => handleClick(1)} />
+      <Text content={'More'} size={'Small'}  className={`button ${selected === 2 ? 'selected' : ''} ${styles.button}`} onClick={() => handleClick(2)}/>
+      </div>
+      </div>
+      
+      <Card className={styles.containerSection}>
+      <table>
```

Comment:
Instead of finding things to correct on your work, I saw things to learn from. Great Work Rukayat!